cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -135,6 +131,27 @@ If sub-domains under your master domain are treated as not trusty in your threat
 
 # Defense In Depth Techniques
 
+## Samesite Cookie Attribute
```

Comment:
Any specific reason on why we are moving this section ahead in "defense in depth" measures?

Reply:
Because it is the easiest one and IMO will add at least the same level of security as other techniques in this section. In addition to that this is true defense in depth because it protect against CSRF on other layer (browser instead of server side).